File: JavaScript/konversi-nilai-angka-ke-huruf.js
const prompt = require("prompt-sync")({ sigint: true });
let nilai = prompt("Masukkan nilai kamu: ");

if (nilai <= 100 && nilai >= 85) {
	console.log(`nilai kamu ${nilai}, kamu mendapat A.`);
} else if (nilai >= 70) {
	console.log(`nilai kamu ${nilai}, kamu mendapat B.`);
} else if (nilai >= 40) {
	console.log(`nilai kamu ${nilai}, kamu mendapat C.`);
} else if (nilai >= 20) {
	console.log(`nilai kamu ${nilai}, kamu mendapat D.`);
} else {
	console.log(`nilai kamu ${nilai}, kamu mendapat E.`);
}

File: Python/konversi-nilai-angka-ke-huruf.py
nilai = int(input("Masukkan nilai kamu: "))

if nilai <= 100 and nilai >= 85:
    print("Nilai kamu " + str(nilai) + ", kamu mendapat A.")
elif nilai >= 70:
    print("Nilai kamu " + str(nilai) + ", kamu mendapat B.")
elif nilai >= 40:
    print("Nilai kamu " + str(nilai) + ", kamu mendapat C.")
elif nilai >= 20:
    print("Nilai kamu " + str(nilai) + ", kamu mendapat D.")
else:
    print("Nilai kamu " + str(nilai) + ", kamu mendapat E.")
